bcda/client/bluebutton.go

```diff
@@ -176,23 +178,54 @@ func (bbc *BlueButtonClient) GetCoverage(beneficiaryID, jobID, cmsID, since stri
 	return bbc.getBundleData(u, jobID, cmsID, nil)
 }
 
-func (bbc *BlueButtonClient) GetExplanationOfBenefit(patientID, jobID, cmsID, since string, transactionTime time.Time, claimsWindow ClaimsWindow) (*models.Bundle, error) {
-	// ServiceDate only uses yyyy-mm-dd
-	const svcDateFmt = "2006-01-02"
-
+func (bbc *BlueButtonClient) GetClaim(mbi, jobID, cmsID, since string, transactionTime time.Time, claimsWindow ClaimsWindow) (*models.Bundle, error) {
 	header := make(http.Header)
 	header.Add("IncludeTaxNumbers", "true")
+
+	mbiHash := HashIdentifier(mbi)
 	params := GetDefaultParams()
-	params.Set("patient", patientID)
+	params.Set("mbi", mbiHash)
 	params.Set("excludeSAMHSA", "true")
 
-	if !claimsWindow.LowerBound.IsZero() {
-		params.Add("service-date", fmt.Sprintf("ge%s", claimsWindow.LowerBound.Format(svcDateFmt)))
+	updateParamWithServiceDate(&params, claimsWindow)
+	updateParamWithLastUpdated(&params, since, transactionTime)
+
+	u, err := bbc.getURL("Claim", params)
+	if err != nil {
+		return nil, err
 	}
-	if !claimsWindow.UpperBound.IsZero() {
-		params.Add("service-date", fmt.Sprintf("le%s", claimsWindow.UpperBound.Format(svcDateFmt)))
+
+	return bbc.getBundleData(u, jobID, cmsID, header)
+}
+
+func (bbc *BlueButtonClient) GetClaimResponse(mbi, jobID, cmsID, since string, transactionTime time.Time, claimsWindow ClaimsWindow) (*models.Bundle, error) {
+	header := make(http.Header)
+	header.Add("IncludeTaxNumbers", "true")
+
+	mbiHash := HashIdentifier(mbi)
```

Comment:
I believe the mbi is already hashed. As seen here:
https://github.com/CMSgov/bcda-app/blob/ee32ebfecbc39aa5057cade34b0b9e22854bf646/bcdaworker/worker/worker.go#L197

Reply:
I thought the `CCLBeneficiary.MBI` was the raw value, as it appears to be getting hashed here as well https://github.com/CMSgov/bcda-app/blob/ee32ebfecbc39aa5057cade34b0b9e22854bf646/bcdaworker/worker/bluebutton.go#L15

Reply:
My bad. I see that now as seen here:
https://github.com/CMSgov/bcda-app/blob/fadeccdc727eb55cfc0d3b4cb0946ec8ddcc6aa6/bcdaworker/worker/worker.go#L169
CCLFBeneficiary.MBI is used. Yes that is the raw, and it is perfectly fine.
But we do have CCLFBeneficiary.BlueButtonID which is hashed. Could we use that? Currently, we hashing twice separately.

Reply:
I thought the `CCLFBeneficiary.BlueButtonID` was the referencing the CCW BeneId and not the MBI.  Our records don't have a BeneID and thus we need to search directly on the MBI.

Reply:
Interesting. Apologies - I did not know that. Then this is 🎉 .